modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/PropertyStep.tsx

```diff
@@ -27,15 +27,15 @@ interface IStepProps extends TGenericComponent {}
 
 const Step: React.FC<IStepProps> = ({onChangeStep}) => {
 	const [properties, setProperties] = useState([]);
+	const {observer, onOpenChange, open} = useModal();
 
 	useEffect(() => {
 		const request = async () => {
 			const response = await fetchProperties();
 			setProperties(response.items);
 		};
 		request();
-	}, []);
-	const {observer, onOpenChange, open} = useModal();
+	}, [properties]);
```

Comment:
We don't need to add `property` as a dependency if it is not used inside useEffect, you can remove it.

Reply:
If I remove it, the properties list will not be reloaded when I create a new property

Reply:
Updated as you suggestion